text/0000-read-only-build-layers.md

```diff
@@ -59,6 +59,8 @@ Updates to the lifecycle so that `build` layers created by buildpacks should be
 
 The lifecycle will have to layerize the layers marked `build` as `true` during the `build` phase of the lifecycle instead of the `export` phase.
 
+The layerized output from the `build` phase of lifecycle could be stored at `/<layers>/@exported` (which should not clash with any buildpack IDs) where `<layers>`. The exporter will just read from this to construct the final image.
```

Comment:
Can we add a example path to make sure I am understanding correctly, an exported build layer would end up at `<layers>/@exported/<buildpack ID>/<layer>.tar`?